tests/smoke/test_notebooks_gpu.py

```diff
@@ -119,8 +119,8 @@ def test_dkn_smoke(notebooks):
     )
     results = pm.read_notebook(OUTPUT_NOTEBOOK).dataframe.set_index("name")["value"]
 
-    assert results["res"]["auc"] == pytest.approx(0.4707, rel=TOL, abs=ABS_TOL)
-    assert results["res"]["acc"] == pytest.approx(0.5725, rel=TOL, abs=ABS_TOL)
+    assert results["res"]["auc"] == pytest.approx(0.5653, rel=TOL, abs=ABS_TOL)
+    assert results["res"]["mean_mrr"] == pytest.approx(0.1629, rel=TOL, abs=ABS_TOL)
```

Comment:
I think we should remove the line of f1 and add 'ndcg@5', 'ndcg@10'

Reply:
ok, let me do it

Reply:
finished. @miguelgfierro